### PR TITLE
Text updates for Microsoft Graph GA

### PR DESCRIPTION
**Pull request by JohnAustin-MSFT**
Updates for code and text changes relating to GA release of the Microsoft.Graph endpoint

### CODE REVIEW

**Comment by iambmelt** [comment]
Please hold off on merging this @JohnAustin-MSFT until we sync with @davechmsft

**Comment by JohnAustin-MSFT** [comment]
Roger that.

File: README.md [hunk 1]
```diff
@@ -133,29 +140,29 @@ After you've built the project you can run it on an emulator or device.
...
-* [`UnifiedContactService`](https://github.com/OfficeDev/O365-Android-Unified-API-Snippets/blob/master/unifiedapi/src/main/java/com/microsoft/unifiedapi/service/UnifiedContactService.java)
-* [`UnifiedEventsService`](https://github.com/OfficeDev/O365-Android-Unified-API-Snippets/blob/master/unifiedapi/src/main/java/com/microsoft/unifiedapi/service/UnifiedEventsService.java)
-* [`UnifiedGroupsService`](https://github.com/OfficeDev/O365-Android-Unified-API-Snippets/blob/master/unifiedapi/src/main/java/com/microsoft/unifiedapi/service/UnifiedGroupsService.java)
-* [`UnifiedMailService`](https://github.com/OfficeDev/O365-Android-Unified-API-Snippets/blob/master/unifiedapi/src/main/java/com/microsoft/unifiedapi/service/UnifiedMailService.java)
-* [`UnifiedUserService`](https://github.com/OfficeDev/O365-Android-Unified-API-Snippets/blob/master/unifiedapi/src/main/java/com/microsoft/unifiedapi/service/UnifiedUserService.java)
+* [`MSGraphContactService`](https://github.com/OfficeDev/O365-Android-Unified-API-Snippets/blob/master/unifiedapi/src/main/java/com/microsoft/unifiedapi/service/MSGraphContactService.java)
```
Comment (iambmelt) [inline README.md]: There seems to be a mix of relative & absolute urls - these will point to the wrong branch / repo if we're on a fork

**Comment by davechmsft** [comment]
Looks good!

**Pull request merged**